pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1919,6 +1919,11 @@ public class ServiceConfiguration implements PulsarConfiguration {
         doc = "The number of bytes before triggering automatic offload to long term storage"
     )
     private long managedLedgerOffloadAutoTriggerSizeThresholdBytes = -1L;
+    @FieldContext(
+            category = CATEGORY_STORAGE_OFFLOADING,
+            doc = "The threshold to triggering automatic offload to long term storage"
+    )
+    private long managedLedgerOffloadTimeThresholdInSeconds = -1L;
```

Comment:
```suggestion
    private long managedLedgerOffloadThresholdInSeconds = -1L;
```

Reply:
Please check all.
I noticed here is using `managedLedgerOffloadThresholdInSeconds`

https://github.com/apache/pulsar/pull/17398/files#diff-a9946a41d139c80164fa362288fdf2ec6a5af5a901213881238e47517ddc0f3bR329

Reply:
@tjiuming I see the config name is `managedLedgerOffloadTimeThresholdInSeconds`. It's better change to `managedLedgerOffloadThresholdInSeconds`

In TopicPolicies, we have `managedLedgerOffloadThresholdInBytes`. Keeping the config name in the same pattern will help users to find by configuration name prefix.